soh/soh/Enhancements/randomizer/context.cpp

```diff
@@ -395,6 +395,500 @@ void Context::ParseHintJson(nlohmann::json spoilerFileJson) {
     CreateStaticHints();
 }
 
+std::map<RandomizerGet, uint32_t> Context::RandoGetToFlag = {
+    { RG_KOKIRI_SWORD,           EQUIP_FLAG_SWORD_KOKIRI },
+    { RG_MASTER_SWORD,           EQUIP_FLAG_SWORD_MASTER },
+    { RG_BIGGORON_SWORD,         EQUIP_FLAG_SWORD_BGS },
+    { RG_DEKU_SHIELD,            EQUIP_FLAG_SHIELD_DEKU },
+    { RG_HYLIAN_SHIELD,          EQUIP_FLAG_SHIELD_HYLIAN },
+    { RG_MIRROR_SHIELD,          EQUIP_FLAG_SHIELD_MIRROR },
+    { RG_GORON_TUNIC,            EQUIP_FLAG_TUNIC_GORON },
+    { RG_ZORA_TUNIC,             EQUIP_FLAG_TUNIC_ZORA },
+    { RG_BUY_DEKU_SHIELD,        EQUIP_FLAG_SHIELD_DEKU },
+    { RG_BUY_HYLIAN_SHIELD,      EQUIP_FLAG_SHIELD_HYLIAN },
+    { RG_BUY_GORON_TUNIC,        EQUIP_FLAG_TUNIC_GORON },
+    { RG_BUY_ZORA_TUNIC,         EQUIP_FLAG_TUNIC_ZORA },
+    { RG_IRON_BOOTS,             EQUIP_FLAG_BOOTS_IRON },
+    { RG_HOVER_BOOTS,            EQUIP_FLAG_BOOTS_HOVER },
+    { RG_GOHMA_SOUL,             RAND_INF_GOHMA_SOUL },
+    { RG_KING_DODONGO_SOUL,      RAND_INF_KING_DODONGO_SOUL },
+    { RG_BARINADE_SOUL,          RAND_INF_BARINADE_SOUL },
+    { RG_PHANTOM_GANON_SOUL,     RAND_INF_PHANTOM_GANON_SOUL },
+    { RG_VOLVAGIA_SOUL,          RAND_INF_VOLVAGIA_SOUL },
+    { RG_MORPHA_SOUL,            RAND_INF_MORPHA_SOUL },
+    { RG_BONGO_BONGO_SOUL,       RAND_INF_BONGO_BONGO_SOUL },
+    { RG_TWINROVA_SOUL,          RAND_INF_TWINROVA_SOUL },
+    { RG_GANON_SOUL,             RAND_INF_GANON_SOUL },
+    { RG_OCARINA_A_BUTTON,       RAND_INF_HAS_OCARINA_A },
+    { RG_OCARINA_C_UP_BUTTON,    RAND_INF_HAS_OCARINA_C_UP },
+    { RG_OCARINA_C_DOWN_BUTTON,  RAND_INF_HAS_OCARINA_C_DOWN },
+    { RG_OCARINA_C_LEFT_BUTTON,  RAND_INF_HAS_OCARINA_C_LEFT },
+    { RG_OCARINA_C_RIGHT_BUTTON, RAND_INF_HAS_OCARINA_C_RIGHT },
+    { RG_SKELETON_KEY,           RAND_INF_HAS_SKELETON_KEY },
+    { RG_GREG_RUPEE,             RAND_INF_GREG_FOUND },
+    { RG_FISHING_POLE,           RAND_INF_FISHING_POLE_FOUND }
+};
```

Comment:
A bit weird to have two types of flags in here.

Reply:
It works because RG is still unique in this, and allows it to be used for multiple case types at the same time, but if it's too confusing, I could separate them into two.